canvas_sdk/commands/base.py

```diff
@@ -35,6 +38,11 @@ def _get_effect_method_required_fields(
     def values(self) -> dict:
         return {}
 
+    @property
+    def coding_filter(self) -> Coding | None:
```

Comment:
i'm still doing some verifying in home-app, but this piece may be completely unnecessary!